src/dials/algorithms/refinement/parameterisation/configure.py

```diff
@@ -790,6 +796,22 @@ def build_prediction_parameterisation(
         A prediction equation parameterisation object
     """
 
+    # Set overall default interval_width_degrees values, if requested
+    if options.interval_width_degrees:
+        options.beam.smoother.interval_width_degrees = options.interval_width_degrees
```

Comment:
As comment above: I had anticiated this _only_ applying to the crystal parameters. I would further mention right here that we cannot (IIUC) serialise scan varying beam, goniometer and detector parameters -> it is probably not correct to sweep them up in this?

Reply:
We can serialise scan-varying models other than crystals, but `dials.integrate` does not use them properly.

Reply:
actually, I _might_ have fixed that at least for scan-varying beams. IIRC it is still problematic because things like the mosaicity model ignore the scan-varying parameters. So, while prediction in integration is scan-varying, the overall model remains global